app/i18n/ko-KR/scenes.json

```diff
@@ -28,5 +28,8 @@
   "Your media files are currently being synced with the cloud. ": "현재 미디어 파일들이 클라우드에 동기화되고 있습니다. ",
   "It is recommended that you wait until this finishes before going live.": "방송하기 전에 작업이 끝날 떄까지 기다리는 것을 권장합니다.",
   "Go Live Anyway": "무시하고 방송하기",
-  "Wait": "기다리기"
+  "Wait": "기다리기",
+  "Add a new Scene.": "장면을 추가합니다.",
+  "Remove Scene": "장면을 제거합니다.",
+  "Open the Scene Transitions.": "장면의 변환을 엽니다."
```

Comment:
Unfortunately submitting these translations here won't work.  You'll have to add them to crowdin, since all files other than english are automatically generated.